### DISCUSSION TITLE
Allow `loguru` to have version higher that 0.5.3

### DISCUSSION THREAD

**Discussion by d-a-bunin** [etna-team/etna#498] [Improvements]
### Description

Currently we have a dependency for loguru:
```
loguru = "^0.5.3"
```

It doesn't allow to install e.g. 0.6 or 0.7 versions.

### Thoughts

- Probably, making it `loguru = ">0.5.3"` don't break anything and it also doesn't look like a breaking change.
- We could set upper version border to be the last version at the time of making a fix.

**Comment by egoriyaa** [comment]
Solved in #599
